.github/workflows/encryption-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Encryption Tests 
+
+on:
+  push
+  #workflow_dispatch: {}
+
+permissions:
+  contents: write
+  pull-requests: write
+  id-token: write
+
+jobs:
+  run-tests:
+    permissions:
+      # required for all workflows
+      security-events: write
+      id-token: write
+      contents: write
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [20]
+        os: [ubuntu-latest]
+        mongodb: [8.0.0]
+    name: Encryption tests
+    env:
+      FORCE_COLOR: true
+    steps:
+      - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
+      - name: Setup node
+        uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
+        with:
+          node-version: latest
+      - name: Install Dependencies
+        run: npm install
+      - name: Install mongodb-client-encryption
+        run: npm install mongodb-client-encryption
+      - name: Set up cluster
+        id: setup-cluster
+        uses: mongodb-labs/drivers-evergreen-tools@master
+        with:
+          version: ${{ matrix.mongodb }}
+          topology: sharded_cluster 
+          auth: auth
+      - name: Run Tests
+        run: npx mocha --exit ./test/encryption/*.test.js
```

Comment:
Usually, we'd add an npm script that runs the tests instead of invoking mocha directly

Reply:
Done! Added a script instead and call that script